app/workers/education_form/templates/5495.erb

```diff
@@ -67,16 +65,6 @@ SSN:   <%= @applicant.veteranSocialSecurityNumber %>
 
 VA File Number: <%= @applicant.vaFileNumber %>
 
-Branch of Service:   <%= @applicant.serviceBranch %>
```

Comment:
This looks like it might be attached to these fields https://staging.vets.gov/education/apply-for-education-benefits/application/5495/applicant/service when the applicant has their own service to report.